Docs/migration-guide-5.x-6.x.md

```diff
@@ -0,0 +1,38 @@
+## 5.x to 6.x Migration Guide
+
+- The Voice SDK has been updated for better Swift interoperability. The `TVO` class prefix has been removed from all Twilio Voice types.
+ 
+ ```swift
+    let connectOptions = ConnectOptions(accessToken: accessToken) { builder in
+        // build connect options using builder
+    }
+    let call = TwilioVoice.connect(options: connectOptions, delegate: self)
+ ```
+ 
+- This release has improved API for CallKit integration. In order to use CallKit with SDK, you must set `ConnectOptions.uuid` or `AcceptOptions.uuid` while making or answering a Call. When `ConnectOptions.uuid` or `AcceptOptions.uuid` is set, it is your responsibility to enable and disable the audio device. You should enable the audio device in `[CXProviderDelegate provider:didActivateAudioSession:]`, and disable the audio device in `[CXProviderDelegate provider:didDeactivateAudioSession:]`.
```

Comment:
Is there any recommendation on what they can set the UUID to? Like a Call SID or should it just be a randomly generated ID?

Reply:
Added code snippets. Let me know know WDYT.